test/models/paymentChannelCreate.ts

```diff
@@ -0,0 +1,210 @@
+import { ValidationError } from 'xrpl-local/common/errors'
+import { PaymentChannelCreate, verifyPaymentChannelCreate } from './../../src/models/transactions/paymentChannelCreate'
+import { assert } from 'chai'
+
+/**
+ * PaymentChannelCreate Transaction Verification Testing
+ *
+ * Providing runtime verification testing for each specific transaction type
+ */
+describe('PaymentChannelCreate Transaction Verification', function () {
+    
+    it (`verifies valid PaymentChannelCreate`, () => {
+        const validPaymentChannel: PaymentChannelCreate = {
+            "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "TransactionType": "PaymentChannelCreate",
+            "Amount": "10000",
+            "Destination": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            "SettleDelay": 86400,
+            "PublicKey": "32D2471DB72B27E3310F355BB33E339BF26F8392D5A93D3BC0FC3B566612DA0F0A",
+            "CancelAfter": 533171558,
+            "DestinationTag": 23480,
+            "SourceTag": 11747
+        }        
+        
+        assert.doesNotThrow(() => verifyPaymentChannelCreate(validPaymentChannel))
+    })
+
+    it (`verifies valid PaymentChannelCreate`, () => {
```

Comment:
These two cases have the same name, it might be good to describe what the differences are. (Ex. 'verifies valid PaymentChannelCreate with no optional fields')